drltrace/drltrace.c

```diff
@@ -161,7 +161,8 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
             dr_fprintf(outf, "<invalid memory>");
             /* Just keep going */
         });
-        dr_fprintf(outf, ")");
+        dr_fprintf(outf, ") thread id: "TIDFMT" and return to "PFX"",
```

Comment:
With ASLR (ignoring libs shared w/ other processes on Windows) moving libs around, the user may have no idea where the address is.  We should not assume it's easy to re-run the program, or require re-running and attaching a debugger on the 2nd run to make sense of the output.

It seems much more usable to do what drcov and drmemtrace use: if inside a module (99+% for non-DGC apps) print a module id + offset instead of an absolute address, and link in drmodtrack and dump a module list to translate the module ids to paths.